### ISSUE TITLE
Analytics service doesn't show Edge Function's and Cron's logs [self-hosted]

### ISSUE THREAD

**Issue by alexandr-bsu** [supabase/supabase#32096]
Every time when  I open Logs tab I get this error message. 
But edge functions works fine. Also I have scheduled cron tasks. I just can't see the logs. API Gateway, Postgres, PostgREST, Auth Storage, Realtime logs open normally.

I use remote server with Ubuntu 24.04 LTS and Self-hosted Supabase version in Docker 
supabase-studio version 20241216-2038e28

1. Go to Logs tab
2. Click on Edge Functions or Cron tabs
3. See error

## Screenshots
![image](https://github.com/user-attachments/assets/3bb4777c-5207-47dc-877c-94f2c898f8d8)
![edge function log error](https://github.com/user-attachments/assets/5265fccc-b8f7-4653-852e-b6974a704d52)

## System information
- OS: Ubuntu 24.04 LTS
- supabase-studio version 20241216-2038e28

**Comment by alexandr-bsu** [comment]
I've just tried to update supabase to latest version and postgres to v15.8.1.020. The problem still remains

**Comment by rohankm** [comment]
i confirm that the issue exists

**Comment by tlovett1** [comment]
I am having the same problem and console.log is also not showing for me in the console.  Not sure if they are related

**Comment by liangzhimy** [comment]
same problem

**Comment by SharkGW** [comment]
Same here, anyone know how to fix this? I have upgraded supabase/edge-runtime to 1.66.3.

**Comment by gionkunz** [comment]
Analytics seems to be completely broken in self-hosted atm

**Comment by ShermanTsang** [comment]
> Analytics seems to be completely broken in self-hosted atm

Yep. I have the same problem. Althong, other logs have no error message on pages , but have no any logs too.

**Comment by Ziinc** [comment]
Closing this issue as this bug is due to a regexp_contains dialect translation bug.
updates will be posted in this issue, closing for consolidation
https://github.com/supabase/supabase/issues/33549
